tests/Unit/Controllers/Store/CentreControllerTest.php

```diff
@@ -72,7 +72,28 @@ public function testItCanDownloadARegistrationsSpreadsheet()
         $data = array_filter(explode(PHP_EOL, $content));
         // Shift the headers off.
         $headers = str_getcsv(array_shift($data));
-        // remap the headers onto each line as keys
+
```

Comment:
DO you alsw want a test for the ordering function?

Reply:
Good call